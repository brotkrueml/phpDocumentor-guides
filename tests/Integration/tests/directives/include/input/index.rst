=========================
Document
=========================

.. include:: comments.rst.txt
