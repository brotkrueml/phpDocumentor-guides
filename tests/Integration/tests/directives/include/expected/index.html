<div class="section" id="document">
    <h1>
        Document
    </h1>
</div>
